easybuild/easyconfigs/n/networkx/networkx-2.0-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
+
+name = 'networkx'
+version = '2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/networkx'
+description = """NetworkX is a Python package for the creation, manipulation, and study of the structure, dynamics,
+ and functions of complex networks."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = ['https://files.pythonhosted.org/packages/source/n/networkx/']
```

Comment:
just use `PYPI_SOURCE` constant here?

Reply:
That did not work for some reason. It has been suggested to use the pythonhosted.org url, which has long term support:
https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/thread/4X2XGKEVXYUE2L7W3LSRTDF56ZN4EILD/

Reply:
@boegel ^

Reply:
It should work, if it doesn't, it's a bug.

There are several recommendations around, and they keep changing, so we started to hide this inside the EasyBuild framework, see https://github.com/easybuilders/easybuild-framework/pull/1749 .

Can you retry with `PYPI_SOURCE`, and let us know what the problem is if it doesn't work for you?

Reply:
OK, with the zip file it works, thx!

Reply:
@smoors There's indeed only a `.zip` available on PyPI for networkx 2.0, no `.tar.gz`, so that probably explains the problem you were seeing...